.travis.yml

```diff
@@ -13,13 +13,21 @@ matrix:
     - env: ACTION=tox
       language: python
       python: '3.6'
+      services:
+        - postgresql
       addons:
-        # TODO - Switch to Postgres 11 once we have rolled out PG 11 in production.
-        postgresql: "9.4"
+        postgresql: "11"
       before_install:
+        - sudo apt-get update
+        - sudo apt-get --yes remove postgresql\*
+        - sudo apt-get install -y postgresql-11 postgresql-client-11
+        - sudo cp /etc/postgresql/{9.6,11}/main/pg_hba.conf
+        - sudo service postgresql restart 11
```

Comment:
Could you add a comment, just on this PR, about what is going on here?

Reply:
Let's just move forward with getting rid of Travis instead, it's terrible

Reply:
> Let's just move forward with getting rid of Travis instead, it's terrible

OK. I guess if we did want to bring back public CI we could always do that with GitHub Actions, which should be easier since it is much more container-oriented.